src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -1284,6 +1435,32 @@ impl PartialEq for Function {
     }
 }
 
+#[derive(
+    Debug, Default, Clone, Copy, Deserialize, Serialize, PartialEq, ts_rs::TS, JsonSchema, FromStr, Display, Bake,
+)]
+#[databake(path = kcl_lib::ast::types)]
+#[ts(export)]
+#[serde(rename_all = "snake_case")]
+#[display(style = "snake_case")]
+pub enum ItemVisibility {
+    #[default]
+    Default,
+    Export,
+}
+
+impl ItemVisibility {
+    fn digestable_id(&self) -> [u8; 1] {
+        match self {
+            ItemVisibility::Default => [0],
+            ItemVisibility::Export => [1],
+        }
+    }
+
+    fn is_default<T: Default + PartialEq>(t: &T) -> bool {
+        t == &T::default()
```

Comment:
This function isn't related to `ItemVisibility` at all -- it doesn't take `self` or use `Self`. Should this just be `matches!(self, Self::Default)` and only work for `ItemVisibility` instead?